rke/k8s_rke_system_images_test.go

```diff
@@ -0,0 +1,27 @@
+package rke
+
+import (
+	"testing"
+
+	"github.com/blang/semver"
+)
+
+func TestLoadRKEDefaultK8sVersions(t *testing.T) {
+	var latest semver.Version
+
+	defaultVersion, _ := semver.Parse(loadRKEDefaultK8sVersions()["default"][1:])
+
+	systemImages := loadK8sRKESystemImages()
+
+	for k := range systemImages {
+		v, _ := semver.Parse(k[1:])
```

Comment:
This might not work for use cases like 2.6 where 1.22.x is available but 1.21 is the default. Would need to continue if minor versions don't match and compare only patch.